### ISSUE TITLE
Fix PHP path for Alfred on Monterey

### ISSUE THREAD

**Issue by skotzko** [MikoMagni/Trello-Workflow-for-Alfred#26]
Because the `/bin/bash` node of the workflow directly calls `php`, the workflow breaks on Monterey with the following error:

`/bin/bash: line 8: php: command not found`

This is true even if the user has manually re-installed `php` via `brew`, per [this guidance](https://www.alfredapp.com/blog/releases/alfred-4-6-ready-for-macos-monterey/). 

The fix I found in the [Alfred forums](https://www.alfredforum.com/topic/17062-using-php-on-monterey-macos-12/#comments) was to add the `PATH` into the top of the script, like so:

Added at top of script (see screenshot):
> `export PATH="/opt/homebrew/bin:/usr/local/bin:${PATH}"`
<img width="778" alt="Screen Shot 2022-01-06 at 10 01 29 PM" src="https://user-images.githubusercontent.com/311226/148499428-d8e709be-25dc-4acc-a347-fb56382781e2.png">

This fixed it, since Alfred's php upgrades don't reach into the script contents themselves.

Can you incorporate this into an updated version of workflow?

**Comment by danajeremy** [comment]
@skotzko
I suppose that this fix is only possible if PHP has been installed via Homebrew?
Since migrating to Monterey, the workflow doesn't work but I don't even have any error at all, is this also the case for you?

**Comment by temple-1** [comment]
Thanks, this helped

**Comment by temple-1** [comment]
@skotzko Thanks this helped.

**Comment by MikoMagni** [comment]
Super late reply, but this is no longer an issue with Alfred. 

Alfred will automatically search for PHP install in the following directories. 

```
/opt/homebrew/bin/php
/usr/local/bin/php
```

Alternatively, you can still define your path in the script.

`export PATH="/opt/homebrew/bin:/usr/local/bin:${PATH}"`

**Comment by greenronny** [comment]
Had same issue and none of the solutions worked for me.
Thanks to ChatGPT I found the issue - 
there was a mismatch between the path where my PHP is located.
So, I used `echo $PATH` in terminal to find out where all the good stuff are. 
and then I've added this line to the "run script" block: 
export PATH="added/the/path/from/terminal/here:$PATH"
Thank you @MikoMagni - your workflow makes my life so much better <3
